.craft.yml

```diff
@@ -0,0 +1,12 @@
+minVersion: "0.15.0"
+github:
+  owner: getsentry
+  repo: snuba-sdk
+releaseBranchPrefix: releases
```

Comment:
I'd drop this and use the default (which is `release` and that's what you use below)